helix-term/src/commands.rs

```diff
@@ -979,6 +979,28 @@ mod cmd {
         doc.format(view.id)
     }
 
+    fn set_indent_style(editor: &mut Editor, args: &[&str], event: PromptEvent) {
+        use IndentStyle::*;
+
+        let style = match args.get(0) {
+            Some(arg) if "tabs".starts_with(&arg.to_lowercase()) => Some(Tabs),
+            Some(arg) if arg.len() == 1 => {
+                let ch = arg.chars().next().unwrap();
+                if ('1'..='8').contains(&ch) {
+                    Some(Spaces(ch.to_digit(10).unwrap() as u8))
+                } else {
+                    None
+                }
+            }
```

Comment:
I think this can be simplified by using `.parse` https://doc.rust-lang.org/std/primitive.str.html#method.parse then validating `.ok()` `< 8`

Reply:
Good call.  Thanks!